vars/infra.groovy

```diff
@@ -272,17 +272,19 @@ void prepareToPublishIncrementals() {
 void maybePublishIncrementals() {
     if (new InfraConfig(env).isRunningOnJenkinsInfra() && currentBuild.currentResult == 'SUCCESS') {
         stage('Deploy') {
-          timeout(15) {
+          timeout(300) {
```

Comment:
not sure whether this is needed, I added it as a safety net in case the connection get's broken to try prevent zombie builds